src/main/java/com/exactpro/th2/crawler/Crawler.java

```diff
@@ -430,21 +595,20 @@ private GetIntervalReport getInterval(Iterable<Interval> intervals) throws IOExc
 
         LOGGER.info("Crawler retrieved {} intervals from {} to {}", intervalsNumber, from, to);
```

Comment:
Maybe print full information about interval (Interval.toString method) additianlly

Reply:
Which interval? We print the aggregated statistics about the current state. We don't have an interval to print it here

Reply:
What about foundInterval, lastInterval variables?

Reply:
The `foundInterval` will be logged at line 587. Regarding the `lastInteval` variable - yes, I think we can log it as well